### DISCUSSION TITLE
Wrapping / Provider

### DISCUSSION THREAD

**Discussion by kamalkech** [solidjs/solid-start#284] [Q&A] [answered]
On solid start, how we can wrap a provider, like in nextjs using file _app.tsx ??

**Comment by ryansolid** [comment]
Not familiar with this mechanism. What do you mean?

**Reply by kamalkech** [reply]
i mean like this :
```tsx
import "../styles/globals.css";
import type { AppProps } from "next/app";

function MyApp({ Component, pageProps }: AppProps) {
  return (
      <Provider client={client}>
          <Component {...pageProps} />
      </Provider>
  )
}

export default MyApp;
```

**Comment by izznat** [comment, marked as answer]
Try adding the `<Provider>` inside `src/root.tsx` file by wrapping the `<Routes>` component. That's the equivalent of `_app.tsx` file from Next.js.

**Reply by kamalkech** [reply]
thank u ;)
